specification/2.0/README.md

```diff
@@ -2183,38 +2150,64 @@ Application-specific data.
 <a name="reference-material"></a>
 ## material
 
-The material appearance of a primitive.
-
-**Properties**
-
-|   |Type|Description|Required|
-|---|----|-----------|--------|
-|**technique**|`string`|The ID of the [`technique`](#reference-technique).|No|
-|**values**|`object`|A dictionary object of parameter values.|No, default: `{}`|
-|**name**|`string`|The user-defined name of this object.|No|
-|**extensions**|`object`|Dictionary object with extension-specific objects.|No|
-|**extras**|`any`|Application-specific data.|No|
-
-Additional properties are not allowed.
-
+glTF 2.0 uses the metal-roughness material model from Physically Based Renderring. The metal-roughness material model is defined by the following properties:
+* Base Color: The base color of the material
+* Metallic-ness: A value between 0 and 1, indicating if the material is metallic (1) or not (0)
+* Roughness: Roughness of the material surface
+
+Todo: Need a good figure illustrating this
+
```

Comment:
Renderring -> Rendering (also appeared above, search for "Renderring")

As for the figure: https://github.com/tsturm/glTF/tree/master/extensions/Vendor/FRAUNHOFER_materials_pbr#metal---roughness could be a start (but maybe a simpler one, or one involving a Khronos/glTF logo would be nicer)